Code/V2/src/components/key.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import theme from '../themes/index';
+
+const Key = (props) => {
+        var keyColor; 
+        switch (props.status) {
+                case 'flashed':
+                        keyColor = theme.key.flashed;
+                        break;
+                case 'selected':
+                        keyColor = theme.key.selected;
+                        break;
+                default:
+                        keyColor = theme.key.neutral;
+
+        }
+
+        const Button = styled.button`
+        font-size: 1em;
+        color: ${theme.key.script};
+        margin: 0.25em;
+        padding: 0.25 0.5em;
+        border-radius: 8px;   // border-radius seems to round the button's edges with larger values and sharpen with smaller
+        background: ${keyColor};   
+        border: 2px solid ${keyColor};
+         `;
+
+        return (<Button>{props.children}</Button>
```

Comment:
Please fix formatting here (ie: place the closing bracket & semi colon on the same line)